src/borg/key.py

```diff
@@ -18,14 +18,20 @@
 from .helpers import Chunk
 from .helpers import Error, IntegrityError
 from .helpers import yes
-from .helpers import get_keys_dir
+from .helpers import get_keys_dir, get_key_nonces_dir
 from .helpers import bin_to_hex
 from .helpers import CompressionDecider2, CompressionSpec
 from .item import Key, EncryptedKey
+from .platform import SaveFile
+from .remote import InvalidRPCMethod
 
 
 PREFIX = b'\0' * 8
+NONCE_SPACE_RESERVATION = 1000
```

Comment:
is that for 1000 * 16 bytes data to encrypt?

guess we rather go to 2**28 or so.